database.py

```diff
@@ -0,0 +1,226 @@
+from sqlalchemy import create_engine, Column, Integer, String, Table, func, inspect
+from sqlalchemy import MetaData
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.orm import mapper
+
+metadata = MetaData()
+dbname = "./Database/data.db"
+engine = create_engine(f'sqlite:///{dbname}')
+
+
+class User(object):
+    id = None
+    name = None
+
+    def __init__(self, name, id):
+        self.name = name
+        self.id = id
+
+
+class KPOP(User):
+    pass
+
+
+class BTS(User):
+    pass
+
+
+class GOT7(User):
+    pass
+
+
+class MonstaX(User):
+    pass
+
+
+class Twice(User):
+    pass
+
+
+class Blackpink(User):
+    pass
+
+
+class ROCK(User):
+    pass
+
+
+class TheBeatles(User):
+    pass
+
+
+class PinkF(User):
+    pass
+
+
+class GreenD(User):
+    pass
+
+
+class Aerosmith(User):
+    pass
+
+
+class LinkinP(User):
+    pass
+
+
+class TaylorSwift(User):
+    pass
+
+
+class BensonBoone(User):
+    pass
+
+
+class EdSheeran(User):
+    pass
+
+
+class Adele(User):
+    pass
+
+
+class Beyonce(User):
+    pass
+
+
+class TimMcGraw(User):
+    pass
+
+
+class PopMusic(User):
+    pass
+
+
+class GeorgeStrait(User):
+    pass
+
+
+class JohnnyCash(User):
+    pass
+
+
+class GarthBrooks(User):
+    pass
+
+
+class CarrieUnderwood(User):
+    pass
+
+
+class Country(User):
+    pass
+
+
+tables = ['George Strait', 'Johnny Cash', 'Garth Brooks', 'Carrie Underwood', 'kpop_bands', 'pop_singers',
+          'country_singers', 'bts_songs', 'blackpink_songs', 'Adele', 'Tim McGraw', 'Beyonce', 'Ed Sheeran',
+          'Benson Boone', 'rock_bands', 'Got7', 'MonstaX_songs', 'GreenDay_songs', 'Aerosmith_songs',
+          'LinkinPark_songs', 'Twice', 'TheBeatles', 'PinkF', 'TaylorSwift']
+classes = [GeorgeStrait, JohnnyCash, GarthBrooks, CarrieUnderwood, KPOP, PopMusic, Country, BTS, Blackpink, Adele,
+           TimMcGraw, Beyonce, EdSheeran, BensonBoone, ROCK, GOT7, MonstaX, GreenD, Aerosmith, LinkinP, Twice,
+           TheBeatles, PinkF, TaylorSwift]
```

Comment:
You don't understand the concept of classes and relational databases.

Reply:
What some series of lectures about the relational database. 
For example, this short video to get the idea https://www.youtube.com/watch?v=NvrpuBAMddw&ab_channel=PrescottComputerGuy

Reply:
Also watch vidoe about Normal Form fro example like this one https://youtu.be/GFQaEYEc8_8